gruntfile.js

```diff
@@ -36,13 +76,74 @@ module.exports = function (grunt) {
                 options: {
                     debug: true
                 }
-            }
+            },
+            "ai-minify": {
+                options: {
+                    debug: true,
+                    //testOnly: true,
+                }
+            },
+            "qunit" : {
+                all: {
+                    options: {
+                    }
+                }
+            },
+            connect: {
+                server: {
+                    options: {
+                        port: 9001,
+                        base: '.'
+                    }
+                }        
+            }            
         };
 
         for (var key in modules) {
             if (modules.hasOwnProperty(key)) {
                 var modulePath = modules[key].path;
                 var moduleCfg = modules[key].cfg;
+                var packageJsonFile = modulePath + '/package.json';
+
+                if (grunt.file.exists(packageJsonFile)) {
+                    // Read the actual module version from the package.json
+                    var pkg = grunt.file.readJSON(modulePath + '/package.json');
```

Comment:
Just adding the minify tasks, referencing the package.json as the utility packages we don't want or need to convert them.